lib/data_fetchers/git_fetcher.rb

```diff
@@ -38,18 +38,31 @@ def line_changes
   end
 
   def languages
-    languages = github.languages.map do |language, percent|
+    languages = github.languages
+      .merge(gitlab.languages) { |key, value1, value2| ((value1 + value2) / 2.0).round }
```

Comment:
Updated.

I'm thinking, that we should maybe change the technologies to only consider the `primary_framework` per repo. What do you think?